src/api/Intelligence.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @flow
+ * @file Helper for the Box AI API endpoint
+ * @author Box
+ */
+
+import Base from './Base';
+import { ERROR_CODE_INTELLIGENCE } from '../constants';
+import type { BoxItem } from '../common/types/core';
+import type { ElementsErrorCallback } from '../common/types/api';
+
+class Intelligence extends Base {
```

Comment:
I see other recent Apis extend `MarkerBasedApi` which extends `Base` , do you think we'd ever need that functionality for Intelligence API?

Reply:
I dont think so